instagram_clone/spec/features/comments_feature_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+feature 'Comments' do
+  before do
+    sign_up
+    do_post
+  end
+
+  context 'create comments' do
+    scenario 'prompt user to fill in a form, then display the comment on the post' do
```

Comment:
Metrics/LineLength: Line is too long. [85/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)